palantir-java-format/src/test/java/com/palantir/javaformat/java/FileBasedTests.java

```diff
@@ -71,6 +75,9 @@ public FileBasedTests(Class<?> testClass, String testDirName) {
                         .isEqualTo(1);
                 String baseName = getNameWithoutExtension(subPath.getFileName().toString());
                 String extension = getFileExtension(subPath.getFileName().toString());
+                if (JAVA_14_TESTS.contains(baseName) && Formatter.getMajor() < 14) {
+                    continue;
+                }
```

Comment:
I'd recommend junit's assumeThat - it can give you a really nice clear 'skipped' thing in the intellij ui

Reply:
Good tip, updated!